src/cocoa/toga_cocoa/widgets/multilinetextinput.py

```diff
@@ -1,6 +1,6 @@
-from travertino.size import at_least
-
+from toga_cocoa.colors import native_color
 from toga_cocoa.libs import *
```

Comment:
At column 1: (F403) 'from toga_cocoa.libs import *' used; unable to detect undefined names